content/updates/index.json

```diff
@@ -1,7 +1,7 @@
 {
   "04-2020": {
-    "title": "🐰 April Serenity updates",
-    "description": "Those are the new updates lo! Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.",
+    "title": "April Serenity updates",
+    "description": "This month is full of new features and updates! Discover our new measurement families, improvements on product associations and on the Connection dashboard. But that's not it! Find out new rules actions and our latest data quality improvements.",
```

Comment:
shouldn't it be 'Connections dashboard', with an S

Reply:
I checked it's called Connection dashboard in the PIM.